src/applications/burial-poc-v6/pages/ReviewPage.jsx

```diff
@@ -0,0 +1,505 @@
+          benefitsUnclaimedRemains: {
+            label:
+              'Are you seeking burial benefits for the unclaimed remains of a Veteran?',
+            value: state.values?.benefitsUnclaimedRemains,
+          },
+        },
+      },
+      {
+        title: 'Benefits Selection: Plot or Interment Allowance',
+        id: 'benefits-plot-allowance',
+        pageUrl: '/benefits/plot-allowance',
+        isShown: state?.values?.benefitsSelection?.plotAllowance === true,
+        fields: {
+          placeOfRemains: {
+            label: 'Place of burial or deceased Veteran’s remains',
+            value: state?.values?.placeOfRemains,
+          },
+          federalCemetery: {
+            label:
+              'Was the Veteran buried in a national cemetery, or one owned by the federal government?',
+            value: state?.values?.federalCemetery,
+          },
+          stateCemetery: {
+            label: 'Was the Veteran buried in a state Veteran’s cemetery?',
+            value: state?.values?.stateCemetery,
+          },
+          govtContributions: {
+            label:
+              'Did a federal/state government or the Veteran’s employer contribute to the burial?  (Not including employer life insurance)',
+            value: state?.values?.govtContributions,
+          },
+          amountGovtContribution: {
+            label: 'Amount of government or employer contribution',
+            value: state?.values?.amountGovtContribution,
+          },
+        },
+      },
+      {
+        title: 'Claimant Contact Information',
+        id: 'claimant-contact-information',
+        pageUrl: '/claimant-contact-information',
+        isShown: true,
+        fields: {
+          claimantAddress: {
+            label: 'Claimant Address',
+            value: {
+              country: {
+                label: 'Country',
+                value: state?.values?.claimantAddress?.country,
+              },
+              street: {
+                label: 'Street',
+                value: state?.values?.claimantAddress?.street,
+              },
+              street2: {
+                label: 'Street 2',
+                value: state?.values?.claimantAddress?.street2,
+              },
+              city: {
+                label: 'City',
+                value: state?.values?.claimantAddress?.city,
+              },
+              state: {
+                label: 'State',
+                value: state?.values?.claimantAddress?.state,
+              },
+              postalCode: {
+                label: 'Postal code',
+                value: state?.values?.claimantAddress?.postalCode,
+              },
+            },
+          },
+          claimantEmail: {
+            label: 'Claimant Email',
+            value: state?.values?.claimantEmail,
+          },
+          claimantPhone: {
+            label: 'Claimant Phone Number',
+            value: state?.values?.claimantPhone,
+          },
+        },
+      },
+      // {
```

Comment:
Just want to make sure, was this intentionally commented out? If so, maybe add an additional comment as to why.